### PR TITLE
libndp: add homepage

### PR DESCRIPTION
**Pull request by skmd0**
**Summary**

- Added homepage to package.yml

**Test Plan**

- Checked if homepage was added. Built package successfully.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/l/libndp/package.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name       : libndp
 version    : '1.6'
-release    : 4
+release    : 5
 source     :
     - https://github.com/jpirko/libndp/archive/v1.6.tar.gz : 565d6c4167f83ec697c762ea002f23e8f0b00828d0749b1ce928f068543e5aad
+homepage   : http://libndp.org/
 license    : LGPL-2.1
```
Comment (malfisya) [inline packages/l/libndp/package.yml:7]: Please fix the license too. It has to follow SPDX format. You can check it [here](https://spdx.org/licenses/)

**Review by malfisya — CHANGES_REQUESTED** [review]
One small thing, other than that looks good

File: packages/l/libndp/package.yml (hunk 1 above)
Comment (skmd0) [reply packages/l/libndp/package.yml]: Done, I amended my commit. Please check if it is ok now :)

**Review by malfisya — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**